### PR TITLE
Upgrade to DataFusion 13.0.0

### PR DESCRIPTION
**Pull request by andygrove**
# Which issue does this PR close?

Closes #.

 # Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: .github/workflows/dev_pr.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
           github.event_name == 'pull_request_target' &&
             (github.event.action == 'opened' ||
              github.event.action == 'synchronize')
-        uses: actions/labeler@4.0.2
+        uses: actions/labeler@v4.0.2
```
Comment (andygrove) [inline .github/workflows/dev_pr.yml:39]: unrelated fix - and this already got merges to master

**Review by avantgardnerio — APPROVED** [review]
